src/transform.py

```diff
@@ -171,6 +187,114 @@ def is_confirmed(self):
     def is_publishable(self):
         return self.is_accepted or self.is_confirmed
 
+    @staticmethod
+    def set_talks_in_parallel(
+        submission: PretalxSubmission, all_sessions: dict[str, PretalxSubmission]
+    ):
+        parallel = []
+        for session in all_sessions.values():
+            if (
+                session.code == submission.code
+                or session.start is None
+                or submission.start is None
+            ):
+                continue
+
+            # If they intersect, they are in parallel
+            if session.start < submission.end and session.end > submission.start:
+                parallel.append(session.code)
+
+        submission.talks_in_parallel = parallel
+
+    @staticmethod
+    def set_talks_after(
+        submission: PretalxSubmission, all_sessions: dict[str, PretalxSubmission]
+    ):
+        # Sort sessions based on start time, early first
+        all_sessions_sorted = sorted(
+            all_sessions.values(), key=lambda x: (x.start is None, x.start)
+        )
+
+        # Filter out sessions
+        remaining_sessions = [
+            session
+            for session in all_sessions_sorted
+            if session.start is not None
+            and session.start >= submission.end
+            and session.code not in submission.talks_in_parallel
```

Comment:
We can change these two lines to include intersecting talks like these:

![image](https://github.com/EuroPython/programapi/assets/75242929/b54e6548-2544-4fda-80a4-7f6a0396d7f8)

In the current version, ``PYO panel`` would not show ``How Python can help monitor governments`` as a *talk after*.

We can change it to do so.